app/controllers/api/v1/lookup_keys_controller.rb

```diff
@@ -58,9 +58,10 @@ def update
 
       def destroy
         if @lookup_key.type == "PuppetclassLookupKey"
-          render_error 'unprocessable_entity', :status => :unprocessable_entity
+          render_message 'Smart class parameters cannot be destroyed',
+            :status => :unprocessable_entity
         else
-          process_response @lookup_key.destroy
+          process_response(@lookup_key.destroy)
```

Comment:
The parentheses you mean? My bad, I tend to have this convention in my head of using parentheses for non-Rails/non-Ruby stdlib functions.